.circleci/config.yml

```diff
@@ -465,9 +465,9 @@ jobs:
     parameters:
       platform:
         type: string
-    executor: << parameters.platform >>_medium
+    executor: << parameters.platform >>_large
     working_directory: << pipeline.parameters.build_dir >>/project
-    parallelism: 4
+    parallelism: 32
```

Comment:
Since you've done the work, but it will be lost to time in a google spreadsheet, can we add a single note to each parallelism change with just one extra datapoint - I'm thinking the difference in time if the parallelism is half of what is specified. (and a link to spreadsheet maybe? But I'm hoping to give one sentence that's enough to have a pretty good idea we're selecting reasonably).

Reply:
@jannotti I think the request is reasonable.  I also prefer to see _why_ configuration values have been picked.

My concern here is the large matrix of factors impacting performance.  In a sentence or two, I think it's difficult to capture a helpful + accurate comment.

Nevertheless, I want to add _something_ because I see value.  What do you think of https://github.com/algorand/go-algorand/pull/4427/commits/fc81e198fe5e30b0caf2df1dd8944cdfdca96d8f?

If it works for you, then let's take it.  If not, I'd revert to proposing we do nothing.  Happy to speak live.

Reply:
Good enough for me